internal/website/content/howto/blob/_index.md

```diff
@@ -9,3 +9,246 @@ providers and accessing them via HTTP. These guides show how to work with
 blobs in the Go CDK.
 
 <!--more-->
+
+## Opening a Bucket {#opening}
+
+The first step in interacting with unstructured storage is connecting to your
+storage provider. Every storage provider is a little different, but the Go CDK
+lets you interact with all of them using the [`*blob.Bucket` type][].
+
+[`*blob.Bucket` type]: https://godoc.org/gocloud.dev/blob#Bucket
+
+The easiest way to open a blob is using [`blob.OpenBucket`][] and a URL
+pointing to the blob, making sure you ["blank import"][] the driver package to
+link it in. See [Concepts: URLs][] for more details. If you need
+fine-grained control over the connection settings, you can call the constructor
+function in the driver package directly (like `s3blob.OpenBucket`). This guide
```

Comment:
driver -> provider again? Also maybe repeat the link to the providers section here, and drop the "This guide ..." sentence.

Reply:
Added the link.